.buildkite/pipeline.yml

```diff
@@ -1,28 +1,31 @@
 steps:
-  - label: Build the docker environment
-    command: make dockerenvbuild
+  - label: Build Python packages
+    command:
+     - make docker-clean
+     - mkdir -p dist
+     - .buildkite/build_whl.sh
 
   - wait
 
-  - label: Build the python packages
-    command: mkdir -p dist && .buildkite/build_whl.sh && docker container prune -f
+  - label: Build Windows installer
+    command: .buildkite/build_windows_installer.sh
 
   - wait
 
-  - label: Build kolibri windows installer
-    command: .buildkite/build_windows_installer.sh
+  - label: Build Debian Packages
+    command: .buildkite/build_debian_pkgs.sh
 
   - wait
 
   - label: Upload artifacts
-    command: .buildkite/setup_and_upload_artifact.sh && docker image prune -f -a
+    command: .buildkite/setup_and_upload_artifact.sh
 
   - label: Sign Windows installer
     command: .buildkite/sign_windows_installer.bat
     agents:
       queue: "windows-sign"
 
-  - block: "Create release artifacts?"
+  - wait
 
-  - label: Build the Android APK file
```

Comment:
Android is a 0.11 roadmap item, please don't remove our ability to build Android. Either refactor it into a docker file, or leave in.

Reply:
See my comment above. It didn't work. I can try to fix that here, or it can be tracked in a different issue for later?

Reply:
As in the APK didn't work? If so, we should leave this code as is, after the block step, and open an issue.

If the build did not complete, then that is a bug introduced by this PR, because it currently completes and produces an APK on current develop: https://buildkite.com/learningequality/kolibri/builds/7330#645e08a0-547b-4e35-b66b-0ef8f3f72890

Reply:
I think it's fine to keep them as a block step, because it keeps our default build times shorter (definitely wanted!), but means they can be generated when people are working specifically on Android.